client_with_ssl.go

```diff
@@ -35,6 +36,7 @@ import (
 //
 // See: https://docs.hazelcast.com/cloud/get-started
 func main() {
+	_ = os.Setenv("HZ_CLOUD_COORDINATOR_BASE_URL", "YOUR_DISCOVERY_URL")
```

Comment:
I just wonder, does `config.Cluster.Cloud` has a property for coordinator base url?

Reply:
Nope, I guess it never got implemented this way before (cc @yuce). I agree that would have been preferable to use instead of environment variables (that we have been using)